api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -616,9 +613,9 @@ public void getConceptByMapping_shouldNotReturnRetiredConcept() {
 		
 	}
 	
-	@Test(expected = APIException.class)
+	@Test
 	public void getConceptByMapping_shouldThrowExceptionIfTwoConceptsHaveSameMapping() {
-		conceptService.getConceptByMapping("127689", "Some Standardized Terminology");
+		assertThrows(APIException.class, ()-> conceptService.getConceptByMapping("127689", "Some Standardized Terminology"));
```

Comment:
please check all the lambdas for whitespaces, so they are in this format `assertThrows(APIException.class, () -> yyy`